crates/ruff_db/src/vendored.rs

```diff
@@ -195,10 +245,10 @@ impl Metadata {
 
 /// Newtype wrapper around a ZipArchive.
 #[derive(Debug)]
-struct VendoredZipArchive(ZipArchive<io::Cursor<&'static [u8]>>);
+struct VendoredZipArchive(ZipArchive<io::Cursor<Cow<'static, [u8]>>>);
```

Comment:
What's the motivation for changing the inner type from `&'static [u8]` to `Cow<&'static, [u8]>` here? Is this so that you can get rid of the `once_cell` dependency in the test code?

Reply:
No, it's so that tests can build an ad-hoc zip file (that doesn't have a `'static` lifetime) but that we can still avoid cloning the vendored data to the heap